src/reducers/agencies.js

```diff
@@ -1,71 +0,0 @@
-import initialData from '../../public/data/agencies-by-state.json'
```

Comment:
yeah, i debated this; for some reason, the action file was named `agency` so i wanted to make them consistent, but i'm happy to make them both be `agencies`; will fix